Translations update from Weblate

Wv5twkFEKh54vo4tta9yu7dHa3 opened the pull request:
Translations update from [Weblate](http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/) for PixelDroid/pixeldroid.

dobios reviewed (commented):
Fixed confilcts please double check

Wv5twkFEKh54vo4tta9yu7dHa3 commented:
> Fix confilcts please

weblate will handle it, I think

Wv5twkFEKh54vo4tta9yu7dHa3 commented:
Or not, maybe merging it broke things

Commit:
Translated using Weblate (Chinese (Simplified))

Currently translated at 88.0% (37 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/zh_Hans/

YoheiZuho committed:
Translated using Weblate (Japanese)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/ja/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/ru/

t3rr0rz0n3 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/ca/

Commit:
Translated using Weblate (Arabic)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/ar/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/fr/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/es/

Commit:
Translated using Weblate (Galician)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/gl/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/fr/

weblate committed:
Translated using Weblate (German)

Currently translated at 90.4% (38 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/de/

wuzhongyi1105 committed:
Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/zh_Hans/

ahangarha committed:
Translated using Weblate (Persian)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/fa/

Commit:
Translated using Weblate (Swedish)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/sv/

Commit:
Translated using Weblate (Basque)

Currently translated at 90.4% (38 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/eu/

t3rr0rz0n3 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/ca/

weblate committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/es/

Commit:
Added translation using Weblate (Bengali (Bangladesh))

Commit:
Translated using Weblate (Bengali (Bangladesh))

Currently translated at 7.1% (3 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/bn_BD/

Commit:
Translated using Weblate (German)

Currently translated at 92.8% (39 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/de/

Commit:
Translated using Weblate (Dutch)

Currently translated at 97.6% (41 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/nl/

veroandi committed:
Added translation using Weblate (Portuguese (Brazil))

veroandi committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/pt_BR/

Commit:
Translated using Weblate (Basque)

Currently translated at 100.0% (42 of 42 strings)

Translation: PixelDroid/pixeldroid
Translate-URL: http://weblate.pixeldroid.org/projects/pixeldroid/pixeldroid/eu/
